packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/index.tsx

```diff
@@ -25,11 +25,12 @@ const SetupDrawer: React.FunctionComponent<SetupDrawerProps> = ({
 }) => {
   const [activeStep, setActiveStep] = useState<number>(0);
 
-  const { framework, isPreviewAvailableForFramework } = useSelector(
+  const { storybook, framework, isPreviewAvailableForFramework } = useSelector(
     (state: SliceMachineStoreType) => ({
       framework: getFramework(state),
       isPreviewAvailableForFramework:
         selectIsPreviewAvailableForFramework(state),
+      storybook: state.environment.env?.manifest.storybook,
```

Comment:
create a selector getStorybookUrl please :) (like the getFramework)

Reply:
It is better because you specify only once where to find the data from your store (the helper) so if something changes in how we store data you don't have to change everywhere 😄